app/assets/javascripts/controllers/storage_system/storage_system_form_controller.js

```diff
@@ -0,0 +1,165 @@
+// copied from plugins\manageiq-ui-classic\app\assets\javascripts\controllers\host\host_form_controller.js
+ManageIQ.angular.app.controller(
```

Comment:
AFAIK angularJS is now deprecated in our codebase and if you're not updating an existing form, you're supposed to implement it in React using MiqFormRenderer (i.e. DDF). You can find plenty of examples under `app/javascript/components/`, basically any folder ending with `form`.

Reply:
@skateman we used a heatmap chart in the storage system form. The heatmap chart is coming from angular-patternfly. It seems that there are no charts at all in the react-patternfly, and charts are coming from  @patternfly/react-charts, which is not added to package.json. In addition, @patternfly/react-charts doesn't have a heatmap chart. 
Can we add @patternfly/react-charts to manageiq-ui-classic package.json? (We probably can use another chart instead of heatmap)

Reply:
cc @himdel angular is your field

Reply:
Do you have a screenshot? Combining charfts and forms doesn't sound like something that will work with DDF.

But yes, adding `@patternfly/react-charts` is OK if it helps you. Or feel free to use the carbon equivalent, if that's better.

Reply:
@himdel , you mean a screenshot of the heatmap?
![image](https://user-images.githubusercontent.com/68283004/93171352-97c5c400-f731-11ea-834f-a87f11f8e442.png)

Reply:
Oh! This is not related to this file at all then?

> `storage_system_form_controller.js`

That angular should not exist, any new forms should be DDF.


But as for the Dashboard heatmap, we currently don't have any react dashboards, so possibly using the angular dashboard is fine for now.

But if you want to go ahead and convert to react, go for it, any angular code in our codebase is considered obsolete, so having a react dashboard implementation is definitely :+1:. (In fact, if your team can convert all the other provider dashboards to react, even better.)

Reply:
:+1: for the reimplementation of dashboards in react, if that fits into your timeframe